### PR TITLE
Started on Drag and Drop

### PR DESCRIPTION
**Pull request by Anmol135**
Drag and Drop from Anmol

### CODE REVIEW

**Comment by Anmol135** [comment]
Just updated the code with some changes that follow the actual drag and drop rule.

**Comment by Anmol135** [comment]
Still need to work on checking the right answers but I think I have the base down.

**Comment by Anmol135** [comment]
The validation part has been done and we have some styling done to the page.

**Comment by Anmol135** [comment]
The project is almost done still need to figure out a way to reset answers.

**Comment by FeXd** [comment]
Screenshot for future reference:

image

File: drag_and_drop/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	<ol class="domande" id="idt-q">		
+	</ol>
+	<div class="container" id="answer">
+		
+	</div>
+	
+
+	<p id="demo"></p>
+	<button id="checkAnswer">Check</button>
+	<!-- <button>Check</button> -->
```
Comment (FeXd) [inline drag_and_drop/index.html:24]: Some weird indentation in this file, and a comment that can be deleted.

File: drag_and_drop/main.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    var dragP;
+    /* Events fired on the drag target */
+
+    document.addEventListener("dragstart", function (event) {
+        // The dataTransfer.setData() method sets the data type and the value of the dragged data
+        // event.dataTransfer.setData("Text", event.target.id);
+        dragP = event.target;
+
+        // Output some text when starting to drag the p element
+        // document.getElementById("demo").innerHTML = "Started to drag the p element.";
```
Comment (FeXd) [inline drag_and_drop/main.js:49]: Be sure to remove commented out code that isn't used anymore.

File: drag_and_drop/main.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    });
+
+    // While dragging the p element, change the color of the output text
+    document.addEventListener("drag", function (event) {
+        document.getElementById("demo").style.color = "red";
+    });
+
+    // Output some text when finished dragging the p element and reset the opacity
+    document.addEventListener("dragend", function (event) {
+        // document.getElementById("demo").innerHTML = "Finished dragging the p element.";
```
Comment (FeXd) [inline drag_and_drop/main.js:62]: Same as above comment, removed commented out code.

File: drag_and_drop/main.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+                document.getElementById('idt-q').innerHTML += ` <li class="question">${idtQuestions[i].question}<div class="droptarget" data-answer="${idtQuestions[i].answer}"></div>`
+          
+        
+        }
+        
+    }
+})
+});
+
+// function showDetails(animal) {
```
Comment (FeXd) [inline drag_and_drop/main.js:144]: Remove commented out function.

**Review by FeXd — CHANGES_REQUESTED** [review]
Overall the functionality of the code is good, I would just like to see some cleanup with indentation and commented out code deleted. Good work.

**Comment by Anmol135** [comment]
Fixed Indentation Issues and took out the commented functions.

**Review by FeXd — APPROVED** [review]

**Pull request merged**